lib/std/os/linux.zig

```diff
@@ -1061,6 +1061,14 @@ pub fn getrusage(who: i32, usage: *rusage) usize {
     return syscall2(SYS_getrusage, @bitCast(usize, @as(isize, who)), @ptrToInt(usage));
 }
 
+pub fn tcgetattr(fd: fd_t, termios_p: *termios) usize {
+    return syscall3(SYS_ioctl, @bitCast(usize, @as(isize, fd)), TCGETS, @ptrToInt(termios_p));
+}
+
+pub fn tcsetattr(fd: fd_t, optional_action: TCSA, termios_p: *const termios) usize {
+    return syscall3(SYS_ioctl, @bitCast(usize, @as(isize, fd)), TCSETS + @bitCast(usize, @as(isize, @enumToInt(optional_action))), @ptrToInt(termios_p));
```

Comment:
`TCSETS + @bitCast(usize, @as(isize, @enumToInt(optional_action)))` is mighty awkward. Lets change the `enum` to be a `u32` and then you won't need any casts here at all? It would still be ABI compatible.